draft-ietf-taps-transport-security.md

```diff
@@ -844,6 +855,70 @@ and the per-message nonce in the clear. Everything else is encrypted.
 
 - An unreliable transport protocol such as UDP.
 
+## OpenVPN
+
+OpenVPN {{OpenVPN}} is a commonly used protocol designed to replace IPsec. A
```

Comment:
nit: Can we say that it's an alternative to IPSec, rather than a replacement? (Paul Wouters commented on this for WireGuard in the secdir review.)

Reply:
Sounds good, will do.